.github/workflows/build-artifact.yml

```diff
@@ -0,0 +1,47 @@
+name: 'Build cf.gov artifact'
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This will run on pushes to main on forks of this repo, that may not have a Docker registry or have S3 credentials configured. Is there some way to limit this action to only run if those things are configured?